Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -64,6 +64,22 @@ class AppStoreConnectService {
 
         return BetaTester(output)
     }
+
+    func deleteBetaTesters(emails: [String]) throws -> [Void] {
+        let betaTestersIds = try emails
+            .map {
+                try GetBetaTesterOperation(
+                    options: .init(id: nil, email: $0)
+                )
+                .execute(with: requestor)
+                .await()
+            }
+            .map { $0.betaTester.id }
```

Comment:
Trying something different (I haven't tried it so let me know if it works for you):

```suggestion
        let requests = emails.map {
            GetBetaTesterOperation(options: .init(id: nil, email: $0)).execute(with: requestor)
        }
        
        let ids = try ConcatMany(requests).awaitMany().map(\.betaTester.id)
```